meetupviewmodel/src/main/java/com/gdgistanbul/viewmodel/MeetupViewModel.kt

```diff
@@ -10,19 +11,30 @@ import kotlinx.coroutines.launch
 
 class MeetupViewModel(private val meetupRepo: MeetupRepo) : ViewModel() {
 
-
-    val eventsLiveData = MutableLiveData<List<Event>>()
-    val toastLiveData = MutableLiveData<String>()
+    private val _eventsLiveData = MutableLiveData<List<Event>>()
+    private val _toastLiveData = MutableLiveData<String>()
```

Comment:
I think using singleliveevent will be a more accurate approach.

https://medium.com/androiddevelopers/livedata-with-snackbar-navigation-and-other-events-the-singleliveevent-case-ac2622673150

Reply:
will update this later.